README.md

```diff
@@ -758,6 +786,25 @@ to implement certain methods then derive the remaining methods. Derivations:
     function(other) { return this.lte(other) && other.lte(this); }
     ```
 
+  - [`filter`][] may be derived from [`concat`][], [`empty`][], [`of`][], and
+    [`reduce`][]:
+
+    ```js
+    function(pred) {
+      var F = this.constructor;
+      return this.reduce((f, x) => pred(x) ? f.concat(F.of(x)) : f, F.empty());
+    }
+    ```
```

Comment:
This is how [`Z.filter`][1] is derived.


[1]: https://github.com/sanctuary-js/sanctuary-type-classes/tree/v7.0.0#filter